README.md

```diff
@@ -13,16 +7,69 @@ which has separate instructions.
 - Installs Drupal in a `web` subdirectory
 - Utilizes [composer-installers](https://github.com/composer/installers) and [Drupal Composer Scaffold](https://github.com/drupal/core-composer-scaffold) plugins
 - Can be used as project template for composer [create-project command](https://getcomposer.org/doc/03-cli.md#create-project):
-  ```
-  composer create-project az-digital/az-quickstart-scaffolding:2.5.x-dev azqs --no-interaction --no-dev
-  ```
 
+## Notes
+If you are planning on pushing this site to Pantheon, you should use the
+[Pantheon upstream repository](https://github.com/az-digital/az-quickstart-pantheon) as your scaffolding repo,
+which has separate instructions.
+
+## Choosing the Right Branch for Your Quickstart Project
+In the Arizona Quickstart Scaffolding project, different branches serve various purposes. Here's a guide to understanding which branch you should choose:
+
+### Main Branch (`dev-main`, `2.8.x`)
+
+- **Purpose**: The main branch contains all the latest features and changes. It represents the cutting-edge development but may be less stable than specific release branches. You can use it with `dev-main` or an alias for the latest unreleased version (e.g., `2.8.x`).
```

Comment:
This is not consistent with the current contents of `composer.json` in the main branch, correct?  (Since it curently requires `az-digital/az_quickstart:~2.7`)

Reply:
True, I got a little confused, thinking I was writing about quickstart.

Reply:
I guess that is what you were talking about in the morning meeting when you were saying that we should change that to the dev-main alias.

I think with the explanation in these docs, we could switch to using the dev-main alias.